src/main/webapp/WEB-INF/pages/inc/menu.ftl

```diff
@@ -61,8 +61,7 @@
                         <li class="nav-item">
                             <a class="nav-link [#if currentMenu=='admin']active[/#if]" href="${baseURL}/admin/">[@s.text name="menu.admin"/]</a>
                         </li>
-                    [/#if]
-                    [#if !adminRights && !managerRights]
+                    [#else]
```

Comment:
In this commit was removed target="_blank" in line 85

Reply:
Done.

Reply:
There is no need to make changes for this files:
    src/main/webapp/WEB-INF/pages/faq.ftl
    src/main/resources/ApplicationResources_es.properties